pom.xml

```diff
@@ -29,7 +44,7 @@
           <artifactId>spring-boot-starter-logging</artifactId>
         </exclusion>
       </exclusions>
-    </dependency> 
+    </dependency>   
```

Comment:
redundant space, I know it is not part of this PR ... But we must have a story that runs linter/checkstyle as part of the CI

Reply:
done